src/component/MembersList.jsx

```diff
@@ -21,6 +21,39 @@ const MembersList = ({teamId}) => {
       });
   }, []);
 
+  const toggleMemberSelection = (memberId) => {
+    setSelectedMembers((prevSelected) => {
+      if (prevSelected.includes(memberId)) {
+        return prevSelected.filter(id => id !== memberId);
+      } else {

```

Comment:
When you return within an 'if' statement, you do not need to include an 'else' statement.